app/src/main/java/com/example/playlistmaker/mediateka/MediatekaScreen.kt

```diff
@@ -0,0 +1,199 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.playlistmaker.R
+import com.example.playlistmaker.createPlaylist.domain.model.Playlist
+import com.example.playlistmaker.mediateka.favorite.ui.FavoriteScreen
+import com.example.playlistmaker.mediateka.favorite.ui.FavoriteState
+import com.example.playlistmaker.mediateka.favorite.ui.FavoritesTracksViewModel
+import com.example.playlistmaker.mediateka.playlist.ui.PlaylistScreen
+import com.example.playlistmaker.mediateka.playlist.ui.PlaylistState
+import com.example.playlistmaker.mediateka.playlist.ui.PlaylistViewModel
+import com.example.playlistmaker.player.domain.model.Track
+import kotlinx.coroutines.launch
+
+@Composable
+fun MediatekaScreen(
+    viewModelFavorite: FavoritesTracksViewModel,
+    viewModelPlaylist: PlaylistViewModel,
+    onClickCreatePlaylist: () -> Unit,
+    onClickCurrentPlaylist: (playlist: Playlist) -> Unit
+) {
+
+    val stateFavorite by viewModelFavorite.state.collectAsState()
+    val statePlaylist by viewModelPlaylist.state.collectAsState()
+
+    MediatekaScreen(
+        stateFavorite = stateFavorite,
+        statePlaylist = statePlaylist,
+        onClickCreatePlaylist = onClickCreatePlaylist,
+        onClickCurrentPlaylist = onClickCurrentPlaylist,
+        onClickPlayer = viewModelFavorite::onClick
+    )
+
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun MediatekaScreen(
+    stateFavorite: FavoriteState,
+    statePlaylist: PlaylistState,
+    onClickCreatePlaylist: () -> Unit,
+    onClickCurrentPlaylist: (playlist: Playlist) -> Unit,
+    onClickPlayer: (track: Track) -> Unit
+) {
+
+    val scope = rememberCoroutineScope()
+    val pagerState = rememberPagerState(pageCount = { 2 })
+    val selectedTabIndex = remember { pagerState.currentPage }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(horizontal = 16.dp)
+            .padding(top = 14.dp)
+    ) {
+        Text(
+            text = stringResource(id = R.string.mediateka),
+            fontFamily = FontFamily(Font(R.font.ys_display_medium)),
+            fontSize = 22.sp,
+            color = if (isSystemInDarkTheme()) colorResource(id = R.color.white) else colorResource(
+                id = R.color.black
+            ),
```

Comment:
Лучше использовать скобки, иначе ухудшается читаемость кода

```suggestion
color = if (isSystemInDarkTheme()) {
                colorResource(id = R.color.white)
            } else {
                colorResource(
                    id = R.color.black
                )
            }
```

Reply:
Сделано